### PR TITLE
rev version to 2.11.4 for release

### PR DESCRIPTION
**Pull request by davidmrdavid**
As in title, routine PR

### CODE REVIEW

**Review by nytian — APPROVED** [review]

**Comment by davidmrdavid** [comment]
Related: https://github.com/microsoft/durabletask-netherite/pull/313

File: test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,11 @@ RUN cd /root/test/SmokeTests/BackendSmokeTests/Netherite && \
     dotnet publish -c Release --output /home/site/wwwroot
 
 # Deploy the app
-FROM mcr.microsoft.com/azure-functions/dotnet:4.0
+# TODO: For some reason, the 'floating tag' `dotnet:4.0` does not contain
+# the latest Host. Our Netherite tests will fail without an up to date host.
+# Therefore, we're pinning to a more up to date tag than 4.0, but we should
+# eventually move to a 'floating tag' again.
+FROM mcr.microsoft.com/azure-functions/dotnet:4.22.0
```
Comment (davidmrdavid) [inline test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile:17]: @bachuv: I'm told the following edit could to get the "True" latest Docker image. Might trying it out? If it fails, you can revert back.

```suggestion
FROM mcr.microsoft.com/azure-functions/dotnet:4
```

Comment (bachuv) [reply test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile:17]: Yes, I'll merge this change right now.

Comment (bachuv) [reply test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile]: The Netherite smoke test passed with the change!

File: test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile [hunk 2]
```diff
@@ -10,7 +10,11 @@ RUN cd /root/test/SmokeTests/BackendSmokeTests/Netherite && \
     dotnet publish -c Release --output /home/site/wwwroot
 
 # Deploy the app
-FROM mcr.microsoft.com/azure-functions/dotnet:4.0
+# TODO: For some reason, the 'floating tag' `dotnet:4.0` does not contain
+# the latest Host. Our Netherite tests will fail without an up to date host.
+# Therefore, we're pinning to a more up to date tag than 4.0, but we should
+# eventually move to a 'floating tag' again.
```
Comment (davidmrdavid) [inline test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile:16]: We can remove this TODO comment now
```suggestion
```

**Review by bachuv — APPROVED** [review]

**Pull request merged**